### ISSUE TITLE
'Blog selected text' doesn't work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#313]
```
What's the problem?

When I 'blog selected text' in Chrome, the quote comes in as a link, not a 
blockquote. Please fix it so it works the same as in Firefox!

What browser are you using?

12.0.742.4 canary build

What version of ScribeFire are you running?

1.5.1
```

-----
Original issue reported on code.google.com by `t...@e1evation.com` on 23 Apr 2011 at 3:16

**Comment by GoogleCodeExporter** [comment]
```
Same is true with the most recent version of Chrome Stable...
```

-----
Original comment by `t...@e1evation.com` on 23 Apr 2011 at 3:39

**Comment by GoogleCodeExporter** [comment]
```
I never got a reply to this...
```

-----
Original comment by `toddlohe...@gmail.com` on 14 Jul 2011 at 8:58

**Comment by GoogleCodeExporter** [comment]
```
Fixed in r313.
```

-----
Original comment by `cfinke` on 19 Aug 2011 at 2:10
* Changed state: **Fixed**
* Added labels: **Milestone-1.9**

**Comment by GoogleCodeExporter** [comment]
```
Thanks. How do I get this into Chrome or does it happen automatically?
```

-----
Original comment by `toddlohe...@gmail.com` on 19 Aug 2011 at 2:15

**Comment by GoogleCodeExporter** [comment]
```
I'm using version 1.8 now and the problem persists. It's not fixed yet...
```

-----
Original comment by `toddlohe...@gmail.com` on 19 Aug 2011 at 12:06

**Comment by GoogleCodeExporter** [comment]
```
It will be included in the version 1.9 update.
```

-----
Original comment by `cfinke` on 19 Aug 2011 at 1:53

**Comment by GoogleCodeExporter** [comment]
```
When does that happen?
```

-----
Original comment by `toddlohe...@gmail.com` on 19 Aug 2011 at 2:02
